presentation/src/main/java/com/example/presentation/dashboard/adapter/DashboardCurrencyPairUi.kt

```diff
@@ -21,6 +24,14 @@ interface DashboardCurrencyPairUi {
             currencyPairText.changeText(currencyPair)
             ratesTextView.changeText(rates)
         }
+
+        override fun delete(viewModel: ClickActions) {
+            val dividerIndex = currencyPair.indexOf(" / ")
```

Comment:
val parts = currencyPair.value().split("/")
val from = parts[0]
val to = parts[1]